### PR TITLE
Archive Holding as Array

### PR DESCRIPTION
**Pull request by csmith-he**
By switching Archive Holding from a multiple instance to single instance card, the logic behind the report section for Archive Holding failed to work.

If a value is returned for Archive Holding, it is forcing into being an array.

Index value has been removed for Holding subheading in the template as, in Keystone, this should only have one value.

### CODE REVIEW

File: arches_her/templates/views/components/reports/archive-source.htm [hunk 1]
```diff
@@ -71,15 +71,15 @@ <h2 class="aher-report-section-subtitle" style="display: block; margin-bottom: 5
                     <!-- ko foreach: { data: archiveHolding, as: "archive", noChildContext: true} -->
                     <!-- ko if: !!archive && archive.data -->
                     <div class="aher-report-subsection">
-                        <h2 class="aher-report-section-subtitle"><i data-bind="click: function() {toggleVisibility(archive.visible)}, css: {'fa-angle-double-right': archive.visible(), 'fa-angle-double-up': !archive.visible()}" class="fa toggle"></i> <span>{% trans "Holding" %}</span> <span data-bind="text: $index() + 1"></span></h2>
+                        <h2 class="aher-report-section-subtitle"><i data-bind="click: function() {toggleVisibility(archive.visible)}, css: {'fa-angle-double-right': archive.visible(), 'fa-angle-double-up': !archive.visible()}" class="fa toggle"></i> <span>{% trans "Holding" %}</span></h2>
```
Comment (aj-he) [inline arches_her/templates/views/components/reports/archive-source.htm:74]: @csmith-he is this H2 even needed now? It is actually not accessible as there are child H2 elements in the iterator so best removing it.

**Review by aj-he — CHANGES_REQUESTED** [review]
Just remove the "Holding" H2 and I think that will sort it!

File: arches_her/templates/views/components/reports/archive-source.htm (hunk 1 above)
Comment (csmith-he) [reply arches_her/templates/views/components/reports/archive-source.htm]: @aj-he - I've now removed this line

**Review by aj-he — APPROVED** [review]
Looks good!

**Pull request merged**